easybuild/easyconfigs/f/ffnvcodec/ffnvcodec-12.1.14.0.eb

```
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'ffnvcodec'
+version = '12.1.14.0'
+
+homepage = 'https://git.videolan.org/?p=ffmpeg/nv-codec-headers.git'
+
+description = """FFmpeg nvidia headers. Adds support for nvenc and nvdec. Requires Nvidia GPU and drivers to be present
+(picked up dynamically)."""
+
+toolchain = SYSTEM
+
+sources = [{
+    'git_config': {
+        'url': 'https://git.videolan.org/git/ffmpeg/',
+        'repo_name': 'nv-codec-headers',
+        'tag': 'n%(version)s',
+    },
+    'filename': SOURCE_TAR_GZ,
+}]
+checksums = ['a7da1532e61dee98c2a917a7a3f9127b94a1d72d44aa1cce23e3d797b114d0c0']
```

```suggestion
checksums = [None]
```

As it is a git download.